### PR TITLE
ft: add star ratings component [#44]

### PR DESCRIPTION
**Pull request by SergeInGithub**
# Description

This will add the following component:
- Product ratings star

The mentioned card will be show in various pages such as viewing a specific product's review, viewing all products, etc...

⚠️ Please note that the eslint-disable lines are only mere temporary solutions because the component is not being called anywhere(for now) but rather being presented through storybook.

📹   Loom video link: https://www.loom.com/share/1635d98c1d214c74a5c48fcd9cefc97f

Fixes # (#44)

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have created a loom video for the new feature or enhancement
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by musllim — CHANGES_REQUESTED** [review]
remove the shadow on stars container

File: src/components/ui/Stars.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable react/require-default-props */
```
Comment (mucyoalexaxel) [inline src/components/ui/Stars.tsx:2]:
```suggestion
```
fix this es-lint errors.

File: src/components/ui/index.ts
Comment (mucyoalexaxel) [inline src/components/ui/index.ts:1]: @SergeInGithub rename the `ui` folder to `shared`

**Review by mucyoalexaxel — CHANGES_REQUESTED** [review]
@SergeInGithub Great work on this PR, check the suggested changes below.

**Review by shyakadavis — APPROVED** [review]
LGTM

**Review by emilenfc — APPROVED** [review]
LGTM

**Review by musllim — APPROVED** [review]
Looks good

**Review by mucyoalexaxel — APPROVED** [review]
@SergeInGithub great work here

**Pull request merged**